tools/reviewer/service/CodeReviewService.java

```diff
@@ -131,11 +145,11 @@ public void getFile(FileRequest req, StreamObserver<FileResponse> responseObserv
   }
 
   @Override
-  public void createDiff(
-      CreateDiffRequest req, StreamObserver<CreateDiffResponse> responseObserver) {
+  public void createDiff(CreateDiffRequest req, StreamObserver<Empty> responseObserver) {
     FirestoreClient client =
         new FirestoreClient(authService.getProjectId(), authService.getToken());
     client.createDocument(firestoreReviewRoot.get(), req.getDiff());
+    responseObserver.onNext(Empty.newBuilder().build());
```

Comment:
s/newBuilder().build()/getDefaultInstance()

Reply:
@oferb that's another thing that could be enforced in CI if we had it 🙂

Reply:
How would we detect it?